### DISCUSSION TITLE
In-game wiki browser functionality

### DISCUSSION THREAD

**Discussion by sentryadam0000345** [blish-hud/Blish-HUD#823] [Module Ideas]
It would be great if we could have some in-game wiki browser so we can wiki things without having to leave the game.

**Comment by lovelydumpling** [comment]
a browser in general would really be useful when sitting in LFG waiting forever

**Comment by Taschenbuch** [comment]
proof of concepts by estreya:

https://discord.com/channels/531175899588984842/599270434642460753/1128261391992762418
https://discord.com/channels/531175899588984842/599270434642460753/1127899123962609664
